invisible_cities/icaros/krmap_functions_test.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import pandas as pd
+from invisible_cities.core.core_functions import in_range, shift_to_bin_centers
+import itertools
+from new_icaromaps import create_NaN_map, med_fun, fit_function, map_3D_fits, merge_maps, include_coordinates, is_even
+from pytest import raises
+
+
+def test_empty_medfun():
```

Comment:
Try to be a bit more descriptive in the test name. Also keep the structure `test_TESTEDFUN_DESCRIPTION`:

```suggestion
def test_medfun_empty_input():
```